job-dsls/jobs/kie/kogito/kogito_tooling_prerelease_branch_UMB_trigger.groovy

```diff
@@ -0,0 +1,101 @@
+/**
+ * Creates job that triggers when a new `<MAJOR>-<MINOR>-<PATCH>-prerelease` branch is
+   pushed to kogito-tooling repository and notifies QE of this by sending and UMB message.
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+def repo = "kogito-tooling"
+def repoBranch = ''
+def ghOrgUnit = "kiegroup"
+def ghAuthTokenId = "kie-ci-token"
+def labelName = "kie-rhel7 && kie-mem16g"
```

Comment:
Do we really need a 16 GB machine for UMB trigger? I would omit the mem requirement so it can run anywhere..

Reply:
Agreed, will fix.